raster/r.tile/testsuite/testrt.py

```diff
@@ -63,8 +63,8 @@ def test_overlap(self):
         )
         self.assertRasterMinMax(
             map=tile,
-            refmin=74.75374,
-            refmax=156.3299,
+            refmin=74.75373,
+            refmax=156.3300,
             msg="tile-000-000 in degrees must be between 74.75374 and 156.3299",
```

Comment:
This needs an update, but actually, deleting should be fine too. The default message will do here. Same below.